Essentials/src/main/java/com/earth2me/essentials/Settings.java

```diff
@@ -1271,7 +1271,7 @@ public long getAutoAfkTimeout() {
     private Set<String> _getAfkTimeoutCommands() {
         final Set<String> timeoutCommands = new HashSet<>();
         for (final String cmd : config.getList("afk-timeout-commands", String.class)) {
-            timeoutCommands.add(cmd.toLowerCase(Locale.ENGLISH));
+            timeoutCommands.add(cmd);
```

Comment:
We should really store a list, not a set, as this will break the expected execution order (and repeated commands, though that's less of a concern)